<launch>
  <!-- State Estimator Node -->
  <arg name="objects" default="UAV0"/>
  <arg name="uav_prefix" default=""/>
  <node pkg="state_estimator" type="state_estimator_node" name="state_estimator" output="screen">
    <rosparam command="load" file="$(find state_estimator)/config/params.json" />
    <param name="uav_prefix" value="$(arg uav_prefix)" />
  </node>
</launch>
